src/wasm/asm-wasm-builder.cc

```diff
@@ -524,6 +527,7 @@ class AsmWasmBuilderImpl : public AstVisitor {
   Isolate* isolate_;
   Zone* zone_;
   TypeCache const& cache_;
+  int depthInFunction_;
```

Comment:
I was thinking depthInFunction_ and a stack of depthInLoop_ (after I implement loops, break, continue). Can also do block_depth and block_depth_in_loop.
Btw, what is the convention around using camel case vs snake case?